web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/CopyData.js

```diff
@@ -70,7 +70,7 @@ export default class CopyData {
   stringQuoteCell(value, column) {
     if (value && _.isObject(value)) {
       value = this.CSVOptions.quote_char + JSONBigNumber.stringify(value) + this.CSVOptions.quote_char;
-    } else if (value && column.cell != 'number' && column.cell != 'boolean') {
+    } else if (value && column.cell != 'number' && column.cell != 'boolean' && column.cell != 'Json') {
```

Comment:
A JSON is still a string. I am not sure if this will work with CSV (when multiple cells are copied). Instead, we should escape the quotes of JSON data when using double quotes.